src/datatype.ts

```diff
@@ -12,6 +12,20 @@ import {ISelectAble, SelectAble} from './idtype';
 import {all, none, Range1D, Range1DGroup, composite, Range} from './range';
 
 /**
+ * TODO
+ */
+export interface IDataDescriptionValue {
```

Comment:
will this interface be implemented by the data types (like IMatrix) or refers it to the optional `value` property within the `IDataDescription`?